chia/full_node/mempool_manager.py

```diff
@@ -173,18 +153,39 @@ def always(mm: MempoolManager, mi: MempoolItem) -> bool:
                     removals.extend(item.removals)
                     additions.extend(item.additions)
                 else:
-                    broke_from_inner_loop = True
-                    break
-        if len(spend_bundles) > 0:
-            log.info(
-                f"Cumulative cost of block (real cost should be less) {cost_sum}. Proportion "
-                f"full: {cost_sum / self.constants.MAX_BLOCK_COST_CLVM}"
-            )
-            agg = SpendBundle.aggregate(spend_bundles)
-            return agg, additions, removals
-        else:
+                    return (spend_bundles, uint64(cost_sum), additions, removals)
+        return (spend_bundles, uint64(cost_sum), additions, removals)
+
+    async def create_bundle_from_mempool(
```

Comment:
this function does not need to be `async`. That's an old issue though, so it's probably best to address it separately.